src/Parser/Warlock/Affliction/Modules/Items/Tier21_2set.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+import ItemDamageDone from 'Main/ItemDamageDone';
+import { formatNumber } from 'common/format';
+import { encodeTargetString } from 'Parser/Core/Modules/EnemyInstances';
+
+import { UNSTABLE_AFFLICTION_DEBUFF_IDS } from '../../Constants';
+
+const TICKS_PER_UA = 4;
+const UA_IDS_SET = new Set(UNSTABLE_AFFLICTION_DEBUFF_IDS);
```

Comment:
My understanding is that for small collections, just keeping this as an array and using `includes` is actually more performant than using a Set.